featuretools/tests/primitive_tests/test_upper_case_word_count.py

```diff
@@ -19,12 +19,11 @@ def test_strings(self):
             dtype="string",
         )
         primitive_func = self.primitive().get_function()
-        answers = pd.Series([2.0, 1.0, 2.0, 3.0, 1.0])
+        answers = pd.Series([2, 1, 2, 3, 1], dtype="Int64")
         pd.testing.assert_series_equal(
-            primitive_func(x),
+            primitive_func(x).astype("Int64"),
```

Comment:
So this means our primitive is not returning the expected data type?

Reply:
Yes, the dtype in the first test is `int64` (I think as there are no null values), and the second test is `object` , which I am looking into

Reply:
I think when we run DFS/CFM, we do cast the columns to match the primitive return_type. So this might be out of scope, and we can keep the `astype`

Reply:
I think I agree. I think the type inference in DFS/CFM is working as all the tests are passing